input/resources/codesystem-au-recorded-sex-or-gender-type.xml

```diff
@@ -0,0 +1,88 @@
+    <title value="AU Recorded Sex or Gender Type" />
+    <status value="active" />
+    <experimental value="false"/>
+    <description value="Codes that represent the type of a Recorded Sex or Gender (RSG) value in an Australian context." />
+    <caseSensitive value="true" />
+    <hierarchyMeaning value="is-a"/>
+    <valueSet value="http://terminology.hl7.org.au/ValueSet/au-recorded-sex-or-gender-type"/>
+    <compositional value="false" />
+    <versionNeeded value="false" />
+    <content value="complete" /> 
+    <concept>
+       <code value="sex-gender"/>
+        <display value="Sex/Gender"/>
+        <definition value="A sex or gender recorded as an interchangeable sex and/or gender value"/>
+    </concept>
+    <concept>
+       <code value="document"/>
+        <display value="Document"/>
+        <definition value="Sex or gender recorded in a document."/>
+    </concept>
+    <concept>
+        <code value="insurance-scheme"/>
+        <display value="Insurance Scheme"/>
+        <definition value="The sex or gender recorded through enrolment in an insurance scheme."/>
+        <property>
+          <code value="notSelectable"/>
+          <valueBoolean value="true"/>
+        </property>
+        <concept>
+            <code value="au-medicare"/>
+            <display value="Australian Medicare"/>
+            <definition value="The sex or gender recorded through enrolment in the Australian Medicare universal healthcare insurance scheme"/>
+        </concept>       
+    </concept>
+    <concept>
+        <code value="registration-and-accreditation-scheme"/>
+        <display value="Registration and accreditation scheme"/>
+        <definition value="The sex or gender recorded through registration in a registration and/or accreditation scheme"/>
+        <property>
+          <code value="notSelectable"/>
+          <valueBoolean value="true"/>
+        </property>
+        <concept>
+            <code value="au-ahpra"/>
+            <display value="Australian Health Practitioner Regulation Agency"/>
+            <definition value="The sex or gender recorded through registration in the Australian Health Practitioner Regulation Agency (AHPRA) registration and accreditation scheme."/>
+        </concept>       
+    </concept>
+    <concept>
+        <code value="digital-health-infrastructure-service"/>
+        <display value="Digital Health Infrastructure Service"/>
+        <definition value="The sex or gender recorded in a digital health infrastructure service"/>
+        <property>
+          <code value="notSelectable"/>
+          <valueBoolean value="true"/>
+        </property>
+        <concept>
+            <code value="au-hi-service"/>
+            <display value="Australian Healthcare Identifiers Service"/>
+            <definition value="The sex or gender recorded in the Australian Healthcare Identifiers (HI) Service"/>
+        </concept>
+        <concept>
+            <code value="au-my-health-record"/>
```

Comment:
Will accept for now but will raise specification feedback. These codes written like natural language will only have a short term life, these systems get renamed, and with a code that very closely resembles the current name of a specific system or scheme it will introduce problems going forwards.. Work needs to be done on this.